<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QComputer</class>
 <widget class="QMainWindow" name="QComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UTComputer</string>
  </property>
  <property name="windowIcon">
   <iconset theme="icon.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="calc">
     <attribute name="title">
      <string>Calculatrice</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>741</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTextEdit" name="vueMessage">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>20</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QTableWidget" name="vueCmd">
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="alternatingRowColors">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTableWidget" name="vuePile">
                 <property name="verticalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="selectionMode">
                  <enum>QAbstractItemView::NoSelection</enum>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>80</number>
                 </attribute>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>50</number>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>30</number>
                 </attribute>
                 <attribute name="verticalHeaderHighlightSections">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>19</number>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QCheckBox" name="checkAnalyser">
                 <property name="text">
                  <string>Afficher l'analyseur de commandes</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Taille de la pile :</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignLeft">
                <widget class="QSpinBox" name="pNb">
                 <property name="maximumSize">
                  <size>
                   <width>80</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>100</number>
                 </property>
                 <property name="value">
                  <number>9</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLineEdit" name="commande">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="suppr">
                 <property name="maximumSize">
                  <size>
                   <width>25</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>C</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="enter">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>50</width>
                   <height>25</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Ok</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="1" column="4">
                <widget class="QPushButton" name="soustraire">
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="4">
                <widget class="QPushButton" name="multiplier">
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QPushButton" name="point">
                 <property name="text">
                  <string>.</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="4">
                <widget class="QPushButton" name="diviser">
                 <property name="text">
                  <string>/</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QPushButton" name="space">
                 <property name="text">
                  <string>_</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="4">
                <widget class="QPushButton" name="additionner">
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="4">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QPushButton" name="evaluer">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>EVAL</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="stocker">
                   <property name="maximumSize">
                    <size>
                     <width>40</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>STO</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="2" column="2">
                <widget class="QPushButton" name="trois">
                 <property name="text">
                  <string>3</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QPushButton" name="cinq">
                 <property name="text">
                  <string>5</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QPushButton" name="six">
                 <property name="text">
                  <string>6</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="quatre">
                 <property name="text">
                  <string>4</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="un">
                 <property name="text">
                  <string>1</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QPushButton" name="neuf">
                 <property name="text">
                  <string>9</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QPushButton" name="deux">
                 <property name="text">
                  <string>2</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QPushButton" name="zero">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="huit">
                 <property name="text">
                  <string>8</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="sept">
                 <property name="text">
                  <string>7</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="bouton" native="true">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QPushButton" name="boutonClavier">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>31</width>
            <height>481</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>500</width>
            <height>650</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="clavierEtendu" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QWidget" name="gridLayoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>271</width>
            <height>481</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QPushButton" name="dup">
             <property name="text">
              <string>DUP</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="lastargs">
             <property name="text">
              <string>LASTARGS</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="dollar">
             <property name="text">
              <string>$</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="lastop">
             <property name="text">
              <string>LASTOP</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="clear">
             <property name="text">
              <string>CLEAR</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="drop">
             <property name="text">
              <string>DROP</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="neg">
             <property name="text">
              <string>NEG</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QPushButton" name="num">
             <property name="text">
              <string>NUM</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QPushButton" name="im">
             <property name="text">
              <string>IM</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="swap">
             <property name="text">
              <string>SWAP</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="den">
             <property name="text">
              <string>DEN</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="re">
             <property name="text">
              <string>RE</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="mod">
             <property name="text">
              <string>MOD</string>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QPushButton" name="redo">
             <property name="text">
              <string>REDO</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="div">
             <property name="text">
              <string>DIV</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="diff">
             <property name="text">
              <string>!=</string>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QPushButton" name="load">
             <property name="text">
              <string>LOAD XML</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="infeg">
             <property name="text">
              <string>=&lt;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="2">
            <widget class="QPushButton" name="sup">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QPushButton" name="egal">
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QPushButton" name="et">
             <property name="text">
              <string>AND</string>
             </property>
            </widget>
           </item>
           <item row="8" column="2">
            <widget class="QPushButton" name="supeg">
             <property name="text">
              <string>&gt;=</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QPushButton" name="inf">
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="ou">
             <property name="text">
              <string>OR</string>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QPushButton" name="non">
             <property name="text">
              <string>NOT</string>
             </property>
            </widget>
           </item>
           <item row="12" column="2">
            <widget class="QPushButton" name="load_test">
             <property name="text">
              <string>LOAD Test</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QPushButton" name="save">
             <property name="text">
              <string>SAVE XML</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Memento :</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Sauvegarde :</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QPushButton" name="undo">
             <property name="text">
              <string>UNDO</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="var">
     <attribute name="title">
      <string>Variables</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="vueVar">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="majAtome">
         <property name="text">
          <string>Mettre à jour les variables</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Programmes</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>731</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Sélectionnez un programme à éditer :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="vueProg"/>
         </item>
         <item>
          <widget class="QPushButton" name="progSaveButton">
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="progCopy">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="progEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
